UI/installer/mp-installer.nsi

```diff
@@ -66,7 +74,14 @@ RequestExecutionLevel admin
 !insertmacro MUI_RESERVEFILE_LANGDLL
 
 Function PreReqCheck
+!ifdef INSTALL64
+	${if} ${RunningX64}
+	${Else}
+		MessageBox MB_OK|MB_ICONSTOP "This version of OBS Studio is not compatible with your system.  Please use the 32bit (x86) installer."
+	${EndIf}
 	; Abort on XP or lower
+!endif
+
 	${If} ${AtMostWinXP}
 		MessageBox MB_OK|MB_ICONSTOP "Due to extensive use of DirectX 10 features, ${APPNAME} requires Windows Vista SP2 or higher and cannot be installed on this version of Windows."
```

Comment:
Does OBS even run on Windows Vista these days? Officially we now only support Windows 8 and above, though it does still run on Windows 7. I think it would be a good time to update this check.

Reply:
In theory, as long as they have SP2 installed. It's definitely not supported, but not sure if it's worth breaking the installer for no good reason.

Reply:
Hmm, maybe a warning instead, that just says it's "Unsupported"?

Reply:
What are we saying is unsupported here though? Who are we sending the warning to? Win7 and older? Vista and older? Right now this specific warning is "will OBS even open". I'm open to being convinced, but I'm not sure that putting restrictions or annoying warnings in the installer is the right approach here, or even worth the effort as I don't think we're going to reach a lot of people trying to install OBS on a Vista SP2 machine or older. This is just a CYA as that OS is too old for OBS to even have a hope of running. Doesn't need to be overly complicated. If it was something we saw often, I'd be more receptive but I don't think this is an issue that needs solving.